dev/RadioButtons/RadioButtons_themeresources.xaml

```diff
@@ -5,13 +5,166 @@
     xmlns:local="using:Microsoft.UI.Xaml.Controls">
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <Thickness x:Key="RadioButtonBorderThemeThickness">1</Thickness>
+            <x:Double x:Key="RadioButtonBorderThemeThickness">1</x:Double>
+            <StaticResource x:Key="RadioButtonForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
```

Comment:
Possibly a dumb question: I'm seeing a whole lot of resources being added here, but I'm not seeing any place where they're actually used.  Am I missing it?

Reply:
Yeah not all of them are used, I thought the pattern was bringing in everything altogether once the control gets touched? But yeah, I can certainly delete the unused once if we don't want them.

Reply:
Are these used elsewhere in the OS repo?  I would've thought that bringing everything over would also involve bringing over the places where they're being used.

Reply:
Oh I see, I should've moved over the templates too, I thought they were already in RadioButton.xaml but looks like they are not.

Reply:
Added now :)

Reply:
Great, thanks!